MAUI/Radial-Menu/Populating-Items.md

```diff
@@ -0,0 +1,723 @@
+    <syncfusion:SfRadialMenu x:Name="radialMenu"
+                    CenterButtonBackFontFamily="Maui Material Assets"
+                    CenterButtonBackFontSize="28"
+                    CenterButtonBackText="&#xe72d;"
+                    CenterButtonFontFamily="Maui Material Assets"
+                    CenterButtonFontSize="28"
+                    CenterButtonText="&#xe710;">
+        <syncfusion:SfRadialMenu.Items>
+            <syncfusion:SfRadialMenuItem FontFamily="Maui Material Assets"
+                            FontSize="20"
+                            Text="&#xe72e;"/>
+            <syncfusion:SfRadialMenuItem  FontFamily="Maui Material Assets"
+                            FontSize="20"
+                            Text="&#xe744;"/>
+            <syncfusion:SfRadialMenuItem FontFamily="Maui Material Assets"
+                            FontSize="20"
+                            Text="&#xe745;"/>
+            <syncfusion:SfRadialMenuItem FontFamily="Maui Material Assets"
+                            FontSize="20"
+                            Text="&#xe73b;"/>
+            <syncfusion:SfRadialMenuItem FontFamily="Maui Material Assets"
+                            FontSize="20"
+                            Text="&#xe762;"/>
+        </syncfusion:SfRadialMenu.Items>
+    </syncfusion:SfRadialMenu>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.RadialMenu;
+
+namespace RadialSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+
+            SfRadialMenu radialMenu = new SfRadialMenu();
+            RadialMenuItemsCollection itemCollection = new RadialMenuItemsCollection()
+            {
+                new SfRadialMenuItem() { FontFamily="Maui Material Assets" FontSize="20" Text="&#xe72e;" },
+                new SfRadialMenuItem() { FontFamily="Maui Material Assets" FontSize="20" Text="&#xe744;" },
+                new SfRadialMenuItem() { FontFamily="Maui Material Assets" FontSize="20" Text="&#xe745;" },
+                new SfRadialMenuItem() { FontFamily="Maui Material Assets" FontSize="20" Text="&#xe73b;" },
+                new SfRadialMenuItem() { FontFamily="Maui Material Assets" FontSize="20" Text="&#xe762;" },
+            };
+            
+            radialMenu.Items = itemCollection;
+            this.Content = radialMenu;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Display font with text in .NET MAUI RadialMenu.](images/populating-items/maui-radialmenu-populating-text-icons.png)
+
+### Adding nested items of radial menu
+
+You can populate the nested levels of items within a menu to group similar actions based on their result. For example, you can group the clipboard operations by adding a clipboard as a main menu and cut, copy, and paste as its children.
+
+The following code snippet demonstrates how to add the nested items of radial menu.
```

Comment:
The following code sample demonstrates how to add the nested items of radial menu.